### PR TITLE
(Preferrably) When Honesty receives a hook that a pull request has been created --OR--, less optimally,  when a developer clicks the create pull request link, make the story as finished in Tracker

### PR DESCRIPTION
**Pull request by kquet**
https://www.pivotaltracker.com/story/show/100331906

### CODE REVIEW

File: lib/app/app.rb [hunk 1]
```diff
@@ -126,6 +126,17 @@ def ignore_branch?(branch)
       #log "[HOOK][RECEIVED_GITHUB_PAYLOAD] /hook.json params: #{JSON.dump(params)}"
```
Comment (alexebird) [inline lib/app/app.rb]: /hook.json and /tracker.json are getting too big.

**Comment by alexebird** [comment]
sorry, I should have done this one first. merge master and re-push this so I can merge it please.

File: lib/app/app.rb (hunk 1 above)
Comment (kquet) [inline lib/app/app.rb]: truf, I have a refactor chore out already.

Comment (alexebird) [inline lib/app/app.rb]: sweet.

**Pull request merged**